eskipfile/remote.go

```diff
@@ -110,7 +122,7 @@ func (client *remoteEskipFile) LoadUpdate() ([]*eskip.Route, []string, error) {
 		if client.verbose {
 			log.Infof("New routes were loaded. New: %d; deleted: %d", len(newRoutes), len(deletedRoutes))
 
-			if client.threshold >= 0 {
+			if client.threshold > 0 {
```

Comment:
This means 0 and negative numbers  disable logging so the "safety guard makes less sense. :)